File: library/src/main/java/com/pengrad/telegrambot/model/request/LabeledPrice.java
package com.pengrad.telegrambot.model.request;

import java.io.Serializable;

/**
 * Stas Parshin
 * 24 May 2017
 */
public class LabeledPrice implements Serializable {
    private final static long serialVersionUID = 0L;

    private String label;
    private Integer amount;

    public LabeledPrice(String label, Integer amount) {
        this.label = label;
        this.amount = amount;
    }
}

File: core/src/commonMain/kotlin/com/pengrad/telegrambot/model/request/LabeledPrice.kt
package com.pengrad.telegrambot.model.request

import java.io.Serializable

/**
 * Stas Parshin
 * 24 May 2017
 */
class LabeledPrice(private val label: String, private val amount: Int?) : Serializable {
    companion object {
        private const val serialVersionUID = 0L
    }
}
